meta-ivi/recipes-graphics/wayland/wayland-ivi-extension_git.bb

```diff
@@ -8,7 +8,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=1f1a56bb2dadf5f2be8eb342acf4ed79"
 PR = "r0"
 
 SRCREV = "736fb654ac81230cf4f9e51a5772d3a02d7639bf"
-SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=http;branch=master \
+SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=http;branch=master;protocol=https \
```

Comment:
The protocol is already http here.

Reply:
Hmm... Seems like meta-ivi set the protocol to `http` [somewhat intentionally](https://github.com/TizenTeam/meta-ivi/commit/d6da283d8d5342c746b1fde7c8e2b73ce89b623f); though the bug they were fixing is on an internal Jira, so I can't say why.

I suppose I'll prefer to leave it `http`, for safety. We're moving to drop meta-ivi anyway.